### ISSUE TITLE
Accessibility: Checkbox next to the mail row is not announced by NVDA

### ISSUE THREAD

**Issue by Murcielago7384** [tutao/tutanota#11303]
### Pre-checks

- [x] This is not a feature request. Existing functionality does not work as intended, **not** missing functionality. I will request features via [discussions](https://github.com/tutao/tutanota/discussions/), [reddit](https://www.reddit.com/r/tutanota/) or [support](https://tuta.com/support)
- [x] I've searched and did not find a similar issue.
- [x] My report does not have personal or private information like email address or recovery code

### App version

v356.260806.1

### Browser

Firefox

### Browser version

LibreWolf version 153.0.3-1

### Operating System

None

### What happened?

Note, while I use a Firefox-based browser as my daily driver, this issue also exists on Chromium-based browsers. The entire context below is also with the understanding that a screen-reader is running.

When logging into TutaMail on a browser,  messages are displayed in such a way that they appear as static text objects on the page, as opposed to headings or links. This makes it extremely difficult to differentiate between where one email b ends and the next begins.
Even while using tab, the experience is sluggish and the screen-reader cannot  identify the control next to each email.

When navigating through the left-hand folder list (inbox, drafts, et cetera), each folder has an "unlabelled" button next to it. This makes it impossible for screen-reader users to identify the purpose of the control without clicking on it.

In terms of email lists, I think a good solution would be to navigate with arrow keys, while in the screen-reader's focus mode, similar to how Outlook on the web handles message lists, if you'd like to see an example. This is (technically) currently possible, but while arrowing down through the message list, NVDA doesn't speak, so it is impossible to know which email is the current focus.

I am happy to give any necessary clarifications in the comments. Tuta's major competitor doesn't seem to care all that much about improving accessibility, so I'd be super grateful to hear your response here. :)

### Other context

Currently running NVDA version 2026.1.1. Free and open source, available from NV Access's website. 
Thanks again.

**Comment by charlag** [comment]
Thank you so much for your feedback! Would you be willing to help us understand some of the issues better?

Regarding the first point, do you mean navigating through the list of emails/conversations or through emails in a single conversation? When I tab through the list items NVDA announces it as "clickable SENDER SUBJECT DATE TAGS". I'm not sure why it doesn't announce what position in the list it is, it could be that the structure is suboptimal to support multiple screen readers (e.g VoiceOver). 

Regarding keyboard navigation in the list, we already have ideas on how to improve that, see https://github.com/tutao/tutanota/issues/10254#issuecomment-3945176041, right now navigating with arrows does not influence focus. We will definitely check how Outlook does it again.

**Comment by charlag** [comment]
Ah and I have created a separate issue about the buttons in the folder list:
https://github.com/tutao/tutanota/issues/11305

**Comment by Murcielago7384** [comment]
Hey there. Thanks so much for the quick response! :)
I'm so happy to hear these improvements are on your radar, and I completely understand that it's not all the work of a minute.
I actually didn't realise before opening the issue that tab navigation worked on the email list, but that is entirely on me rather than an issue your end. I was referring to the email list as opposed to an individual conversation.
The only problem with tab navigation is that NVDA cannot identify the control next to each  "SENDER SUBJECT DATE TAGS", at least not with my set up. I'd be interested to know if you find differently so I know if that's something I need to investigate in my own settings.
I'll stick to tab navigation for now. Thanks again! :)

**Comment by charlag** [comment]
Hi, thank you for clarification!

It is totally understandable that you didn't realize about the tab navigation and I'm glad it works now. The control next to the subject is a selection checkbox. Would be good if it wasn't in the tab index (at least on desktop) but for now we should fix the label. I will update the issue to track that instead :)
